lib/utils/report.mjs

```diff
@@ -282,13 +279,10 @@ function computeEditInfo({ fix, line, result: { stylelint }, ruleName }) {
 		text: node.toString(),
 	});
 
-	// Restore the previous state of the fixed node
-	node.replaceWith(fixedNodeClone);
-
 	// Mark the fixed range as mutated
 	rangesOfComputedEditInfos.push(fixData.range);
 
-	return { ...fixData };
```

Comment:
I think this destructering was a left over from a previous iteration where there were more fields to be assignment.